src/app/components/core/asset-readings/readings-graph/readings-graph.component.html

```diff
@@ -2,29 +2,45 @@
   <div class="modal-background"></div>
   <div class="modal-card">
     <header class="modal-card-head">
-      <p class="modal-card-title is-size-5">{{assetCode}}</p>
-      <button class="delete" aria-label="close" (click)="toggleModal(false)"></button>
+      <p class="modal-card-title is-size-5">{{assetCode}}
+      </p>
+      <div class="close">
+        <button class="delete" aria-label="close" (click)="toggleModal(false)"></button>
+      </div>
     </header>
     <section class="modal-card-body">
       <ng-container *ngIf='showGraph'>
         <div class="columns is-mobile">
           <div class="column is-two-fifths">
-            <div class="select is-small">
-              <select #selectedTime (change)="getTimeBasedAssetReadingsAndSummary(selectedTime.value)">
-                <!-- <option value="0" [selected]="optedTime == 0">None</option> -->
-                <option value="600" [selected]="optedTime == 600">10 minutes</option>
-                <option value="1800" [selected]="optedTime == 1800">30 minutes</option>
-                <option value="3600" [selected]="optedTime == 3600">1 hour</option>
-                <option value="28800" [selected]="optedTime == 28800">8 hours</option>
-                <option value="86400" [selected]="optedTime == 86400">1 day</option>
-              </select>
+            <div id="time-dropdown" class="dropdown is-left">
```

Comment:
I think we should keep the same size, as in develop. And 
click the dropdown, dropdown appears correctly.
close and re-open the modal, the dropdown is open. On close it should have been reset.